deployment/ray/mednist_classifier_ray.ipynb

```diff
@@ -41,7 +41,7 @@
    },
    "outputs": [],
    "source": [
-    "%pip install ray[serve]\n",
+    "!pip install \"ray[serve]\" \"protobuf==3.20.*\"\n",
```

Comment:
Would it be better maintained in the future if we do "pip install ray --no-deps protobuf"? Thanks! @wyli

Reply:
You're right. I figure that `no-deps` is a flag that takes no arguments